backend/battles/admin.py

```diff
@@ -7,5 +7,9 @@ class TeamAdmin(admin.ModelAdmin):
     list_display = ("trainer", "battle")
```

Comment:
![image](https://user-images.githubusercontent.com/23136832/78128187-e41f8080-73eb-11ea-92b3-6d63b37dd674.png)

Is the button supposed to be with this small offset?

Reply:
Is that a frontend question?